docs/tutorial/TODO.md

```diff
@@ -0,0 +1,16 @@
+- [X] 0. Getting ready
+- [X] 1. Create a Scenic app
+- [X] 2. Draw a snake
+- [X] 3. Let the snake move
+- [X] 4. Control snake movement
+- [ ] 5. Add food for the snake
+- [ ] 6. Allow snake to eat
+- [ ] 7. Allow snake to die
+- [ ] 8. Add static score
+- [ ] 9. Add live scoring
+- [ ] 10. Potential later steps
+
+Other ideas:
+
+- move things into components (e.g. the score, the snake)
+- add multiplayer
```

Comment:
I moved the list of the potential chapter into a separated file, just to keep track of the initial idea without polluting the chapters.